shared/domains/chelonia/internals.js

```diff
@@ -46,7 +113,7 @@ sbp('sbp/selectors/register', {
         // if this isn't OP_CONTRACT, get latestHash, recreate and resend message
         if (!entry.isFirstMessage()) {
           const previousHEAD = await sbp('chelonia/out/latestHash', contractID)
-          entry = GIMessage.createV1_0(contractID, previousHEAD, entry.op())
+          entry = GIMessage.createV1_0(contractID, previousHEAD, entry.op(), entry.message().manifest)
```

Comment:
Nothing to change / improve here, but this will conflict with some of my changes.